src/main/index.ts

```diff
@@ -99,6 +101,8 @@ if (app.commandLine.getSwitchValue("proxy-server") !== "") {
   process.env.HTTPS_PROXY = app.commandLine.getSwitchValue("proxy-server");
 }
 
+logger.debug("lprm stuff");
```

Comment:
Maybe change to more descriptive message? Not quite sure what is `lprm` within context.

Reply:
sure, this was meant to be temporary but maybe this logging should stay. I'll add a log prefix as well